src/transformers/configuration_longformer.py

```diff
@@ -25,6 +25,7 @@
 LONGFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
     "longformer-base-4096": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-base-4096/config.json",
     "longformer-large-4096": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-large-4096/config.json",
+    "longformer-large-4096-finetuned-triviaqa": "https://s3.amazonaws.com/models.huggingface.co/bert/allenai/longformer-large-4096-finetuned-triviaqa/config.json",
```

Comment:
stuck to the naming style we also used in Bert for squad finetuned models - hope that's ok @ibeltagy